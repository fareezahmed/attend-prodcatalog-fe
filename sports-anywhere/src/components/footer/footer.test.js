import React from 'react';
import { mount } from 'enzyme'

import Footer from './index';


const setup = (partialProps) => {
    const props = {
        children: "Copyright Â© Fareez Ahmed 2014",
        ...partialProps,
      }
    const wrapper = mount(<Footer { ...props } />);
    return {
        wrapper,
        props,
    }
}

describe('Footer component', () => {
    it('Footer component renders correctly', () => {
        const { wrapper } = setup();
        expect(wrapper).toMatchSnapshot();
    })
})