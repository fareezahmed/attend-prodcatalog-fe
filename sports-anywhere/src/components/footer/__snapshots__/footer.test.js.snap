// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer component renders correctly 1`] = `
<Memo(Footer)>
  <footer>
    <p>
      Copyright Â© Fareez Ahmed 2014
    </p>
  </footer>
</Memo(Footer)>
`;
